lib/hello.js

```diff
@@ -5,7 +5,11 @@ var cors = require('cors');
 function helloRoute() {
   var hello = new express.Router();
   hello.use(cors());
-  hello.use(bodyParser());
+  //hello.use(bodyParser());
```

Comment:
nah, it it removes the deprecate warnings and keeps working fine it's ok to me. This template was already using a most up to date version than other templates (1.15.x vs 1.0.x) that might have the deprecate warnings already, so the other option is to downgrade it to 1.0.2 as the others. But I think it's not needed if the deprecate warnings are gone.